lib/email_processor.rb

```diff
@@ -32,16 +31,20 @@ def process
       topic = Topic.find(ticket_number)
 
       #insert post to new topic
-      message = "Attachments:" if @email.attachments.present? && @email.body.blank?
+      if attachments_enabled?
+        message = "Attachments:" if @email.attachments.present? && @email.body.blank?
```

Comment:
No need to wrap this in an extra if `if attachments_enabled? && @email.attachments.present? && @email.body.blank?` would work.